ui/de-lib/src/main/java/org/iplantc/de/shared/AsyncCallbackWrapper.java

```diff
@@ -50,24 +50,26 @@ private void redirectToLandingPage() {
 
     /**
      * Called whenever a call to the server fails. If the call failed because of an HTTP status code and
-     * that status code represents a redirect request or wasn't recorded then we assume that the user isn't
+     * that status code represents a redirect request or user was unauthorized, then we assume that the user isn't
      * logged in and redirect the user to the login page. The callback that we're wrapping deals with all
      * other errors.
      *
      * @param error the exception or error that indicates why the call failed.
      */
     @Override
     public void onFailure(Throwable error) {
-        if (error instanceof AuthenticationException) {
+       if (error instanceof AuthenticationException) {
+            EventBus.getInstance().fireEvent(new UserLoggedOutEvent());
             LOG.log(Level.SEVERE, "Auth error!!!!!", error);
-            redirectToLandingPage();
```

Comment:
if I'm reading this right, then `redirectToLandingPage()` is private to this file and not used after this PR, so its implementation should probably be removed (along with `LANDING_PAGE` which I believe only it currently cares about)